simulariumio/constants.py

```diff
@@ -63,10 +63,23 @@ class DISPLAY_TYPE(Enum):
     PDB = "PDB"
     OBJ = "OBJ"
     FIBER = "FIBER"
+    SPHERE_GROUP = "SPHERE_GROUP"
     # CUBE = "CUBE"  # coming soon
     # GIZMO = "GIZMO"  # coming soon
 
 
+def SUBPOINTS_FOR_DISPLAY_TYPE(display_type: DISPLAY_TYPE) -> int:
+    """
+    How many subpoints per item saved in subpoints?
+    given the display type of the agent
+    """
+    if display_type == DISPLAY_TYPE.FIBER:
+        return 3
+    if display_type == DISPLAY_TYPE.SPHERE_GROUP:
+        return 4
+    return 1
+
```

Comment:
this is necessary now that subpoints can either contain fiber points or sphere group points and radii

Reply:
add comment about what 3 and 4 mean, or create symbolic constants